express/blocks/new-template-list/new-template-list.js

```diff
@@ -36,11 +36,20 @@ class Masonry {
     this.startResizing = 0;
   }
 
+  static INCOMPLETE = 0;
+
+  static IDENTICAL = 1;
+
+  static CHANGED = 2;
+
```

Comment:
```suggestion
  static INCOMPLETE = 0;
  static IDENTICAL = 1;
  static CHANGED = 2;
```

Reply:
@kptdobe this will result in the folloiwng linting error:
```
  40:3  error  Expected blank line between class members  lines-between-class-members
```

Reply:
ah.. but so much wasted space!